quarkus/tests/integration/src/test/java/org/keycloak/it/storage/map/ChmStorageDistTest.java

```diff
@@ -28,7 +28,7 @@
 import io.quarkus.test.junit.main.LaunchResult;
 
 @RawDistOnly(reason = "Need to check dist path")
-@DistributionTest(reInstall = DistributionTest.ReInstall.BEFORE_TEST)
+@DistributionTest
 public class ChmStorageDistTest {
```

Comment:
Do we need this test now that we're using CHM by default?

Reply:
Not sure if it is enough to assume that CHM is working as expected (config/runtime-wise) as we are covering behaviors that we don't actually check when running tests.